src/main.ts

```diff
@@ -1585,7 +1597,7 @@ export default class WhatsApp {
       await qrcode.toDataURL(`${data.ref},${publicKeyBase64},${this.clientId}`)
     );
 
-    writeFile(this.qrPath, qrCode.data, err => {
+    writeFile(this.parameters.qrPath, qrCode.data, err => {
```

Comment:
```suggestion
    writeFile(this.opts.qrPath, qrCode.data, err => {
```
Update property again